dotnet/ios/WebView/WebView/ViewController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using SafariServices;
+using WebKit;
+
+namespace WebView;
+
+public partial class ViewController : UIViewController
+{
+    private readonly NSUrl url = new ("https://visualstudio.microsoft.com/xamarin/");
```

Comment:
```suggestion
   readonly NSUrl url = new ("https://visualstudio.microsoft.com/xamarin/");
```
It is private by default so this is not necessary